examples/intro-times-and-text.xml

```diff
@@ -0,0 +1,14 @@
+...
+  <body>
+    <div begin="10s" end="13s">
+      <p>
+        <span>A woman climbs into a small sailing boat.</span>
```

Comment:
Can we remove the `<span>` elements, or use them in a meaningful way? Otherwise it gives the wrong impression that spans are always needed.